samples/matter/thermostat/Kconfig

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+mainmenu "Matter Thermostat sample application"
+
+config EXTERNAL_SENSOR
+	bool "Select if the temperature measurement is provided directly from the board or collected from external sensor"
+	help
+	 Set to true to enable the temperature measurement from phisical external sensor, false to mock temperature measurement
+
+config TEMPERATURE_STEP
```

Comment:
As per my previous comment, I'm not sure that we should use generic config names instead of adding some prefixes, such as THERMOSTAT...